target/linux/generic/backport-6.12/610-07-v6.16-net-dsa-b53-do-not-configure-bcm63xx-s-IMP-port-inte.patch

```diff
@@ -33,13 +33,13 @@ Signed-off-by: Paolo Abeni <pabeni@redhat.com>
  {
  	struct b53_device *dev = ds->priv;
 -	u8 rgmii_ctrl = 0, off;
-+	u8 rgmii_ctrl = 0;
- 
+-
 -	if (port == dev->imp_port)
 -		off = B53_RGMII_CTRL_IMP;
 -	else
 -		off = B53_RGMII_CTRL_P(port);
--
++	u8 rgmii_ctrl = 0;
+ 
 -	b53_read8(dev, B53_CTRL_PAGE, off, &rgmii_ctrl);
 +	b53_read8(dev, B53_CTRL_PAGE, B53_RGMII_CTRL_P(port), &rgmii_ctrl);
  	rgmii_ctrl &= ~(RGMII_CTRL_DLL_RXC | RGMII_CTRL_DLL_TXC);
```

Comment:
Why is this change needed?
I do not see any change upstream in this area: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/diff/drivers/net/dsa/b53/b53_common.c?id=v6.12.34&id2=v6.12.33

Reply:
Good catch. [It was already applied](https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/drivers/net/dsa/b53/b53_common.c?id=75f4f7b2b13008803f84768ff90396f9d7553221), so the refresh moved the change to the very similar function `b53_adjust_531x5_rgmii()`.

So in conclusion this patch also needs to be dropped.

Reply:
> Why is this change needed? I do not see any change upstream in this area: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/diff/drivers/net/dsa/b53/b53_common.c?id=v6.12.34&id2=v6.12.33

This block was automatically refreshed. Perhaps it's because the original patch hasn't been completely refreshed? I have tried to revert the change and refresh it again, but got the same result. At least the diff change is actually the same.

Reply:
> Good catch. [It was already applied](https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/drivers/net/dsa/b53/b53_common.c?id=75f4f7b2b13008803f84768ff90396f9d7553221), so the refresh moved the change to the very similar function `b53_adjust_531x5_rgmii()`.
> 
> So in conclusion this patch also needs to be dropped.

No. This patch has not been backported to the 6.12. y branch.